patients/forms.py

```diff
@@ -0,0 +1,8 @@
+from django import forms
+
+class PatientInfoForm(forms.Form):
+    first_name = forms.CharField(label='First Name', max_length=100) #creates html label called first name
+    last_name = forms.CharField(label='Last Name', max_length=100)
+    ohip_number = forms.CharField(label='OHIP Number', max_length=100)

```

Comment:
Does ohip number only consist of ohip number?
Do you not care about expiry dates?

![Ohip Card Example](https://i.imgur.com/vcHRSTa.jpg)